### ISSUE TITLE
Request: add comparison to facrypt

### ISSUE THREAD

**Issue by thomas-mc-work** [rfjakob/gocryptfs-website#15]
Regarding <https://nuetzlich.net/gocryptfs/comparison/>:

A comparison to native ext4 (f2fs is also possible) encryption would be very interesting, as this is a rising solution.

**Comment by rfjakob** [comment]
Good idea. It's somewhat different because you don't have access to the encrypted files. But i guess it's still interesting in terms of performance, as fscrypt should be as fast as it gets.
